gpytorch/module.py

```diff
@@ -159,9 +162,12 @@ def named_constraints(self, memo=None, prefix=""):
         return _extract_named_constraints(module=self, memo=None, prefix="")
 
     def named_variational_parameters(self):
-        for name, param in self.named_parameters():
-            if "variational_" in name:
```

Comment:
hooray 🙌

Reply:
Please do not insult the best 2 lines of code I've ever written.